src/ads.test.ts

```diff
@@ -30,101 +29,122 @@ const renderParagraphs = compose(render, generateParas);
-        const tenParagraphs = renderParagraphs(10)
+    test("Adds second placeholder after 9th paragraph", () => {
+        const tenParagraphs = renderParagraphs(10);
         const tenParagraphsAndTwoAds: any = insertAdPlaceholders(tenParagraphs);
-        expect(tenParagraphsAndTwoAds[10].type).toBe('aside');
+        expect(tenParagraphsAndTwoAds[10].type).toBe("aside");
     });
 });
 
-describe('Adds short classname correctly', () => {
-    test('Adds short classname for articles with less than 15 paragraphs', () => {
-        const fourteenParagraphs = renderParagraphs(14)
-        const fourteenParagraphsAndTwoAds: any = insertAdPlaceholders(fourteenParagraphs);
-        expect(fourteenParagraphsAndTwoAds[3].props.className).toBe('ad-placeholder hidden short');
-        expect(fourteenParagraphsAndTwoAds[10].props.className).toBe('ad-placeholder hidden short');
+describe("Adds short classname correctly", () => {
+    test("Adds short classname for articles with less than 15 paragraphs", () => {
+        const fourteenParagraphs = renderParagraphs(14);
+        const fourteenParagraphsAndTwoAds: any = insertAdPlaceholders(
+            fourteenParagraphs
+        );
+        expect(fourteenParagraphsAndTwoAds[3].props.className).toBe(
+            "ad-placeholder hidden short"
+        );
+        expect(fourteenParagraphsAndTwoAds[10].props.className).toBe(
+            "ad-placeholder hidden short"
+        );
     });
 
-    test('Does not add short classnames for articles with 15 paragraphs', () => {
-        const fifteenParagraphs = renderParagraphs(15)
-        const fifteenParagraphsAndTwoAds: any = insertAdPlaceholders(fifteenParagraphs);
-        expect(fifteenParagraphsAndTwoAds[3].props.className).toBe('ad-placeholder hidden');
-        expect(fifteenParagraphsAndTwoAds[10].props.className).toBe('ad-placeholder hidden');
+    test("Does not add short classnames for articles with 15 paragraphs", () => {
+        const fifteenParagraphs = renderParagraphs(15);
+        const fifteenParagraphsAndTwoAds: any = insertAdPlaceholders(
+            fifteenParagraphs
+        );
+        expect(fifteenParagraphsAndTwoAds[3].props.className).toBe(
+            "ad-placeholder hidden"
+        );
+        expect(fifteenParagraphsAndTwoAds[10].props.className).toBe(
+            "ad-placeholder hidden"
+        );
     });
 });
 
-describe('Handles different DOM structures', () => {
-    test('Does not count other tag types', () => {
+describe("Handles different DOM structures", () => {
+    test("Does not count other tag types", () => {
         const text = renderTextElement([
-            '<p></p>',
-            '<p></p>',
-            '<div></div>',
-            '<div></div>',
-            '<div></div>',
-            '<div></div>',
+            "<p></p>",
+            "<p></p>",
+            "<div></div>",
+            "<div></div>",
+            "<div></div>",
+            "<div></div>",
```

Comment:
Im assuming you had to change what was in the codebase before applying the new linting rules for ' > "

Reply:
`eslint --fix` was able to convert all the single quotes into `"`

Reply:
if you use https://www.npmjs.com/package/@guardian/prettier too these will all revert to `'`